doc/designs/ticket-lifetime-jitter.md

```diff
@@ -0,0 +1,14 @@
+# Ticket Lifetime Jitter
+
+## Overview
+
+Ticket lifetimes can be jittered so that renewals / re-issues do not overwhelm the KDC at a certain moment. 
+The feature is enabled automatically so we can avoid triggering an LDAP query on every AS_REQ and TGS_REQ. 
```

Comment:
I'm not sure this is quite right anymore.  The spirit is correct (i.e., that it's more expensive to check if we should jitter rather than just jitter), but I think in our discussion we decided it doesn't save an LDAP query.

Reply:
True, I hadn't read this document in a while! I removed the line in the upcoming commit.